scipy/linalg/decomp.py

```diff
@@ -7,6 +7,7 @@
 # additions by Bart Vandereycken, June 2006
 # additions by Andrew D Straw, May 2007
 # additions by Tiziano Zito, November 2008
+# additions by Alex Papanicolaou, August 2016
```

Comment:
You don't necessarily need to add a comment here --- who modified what is kept track of by the version contorl.
In particular, the file has been modified by many people since 2008.